efsity/src/main/java/org/smartregister/command/DeleteDataCommand.java

```diff
@@ -0,0 +1,97 @@
+package org.smartregister.command;
+
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+import java.io.*;
+
+
+import org.apache.http.HttpResponse;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.client.methods.HttpDelete;
+
+import org.smartregister.util.FctUtils;
+import picocli.CommandLine;
+
+@CommandLine.Command(name = "delete")
+public class DeleteDataCommand implements Runnable {
+  @CommandLine.Option(
+    names = {"-u", "--baseUrl"},
+    description =
+      "FHIR base URL example http://localhost:8888/fhir",
+    required = true)
+  String baseUrl;
+
+  @CommandLine.Option(
+    names = {"-r", "--resourceType"},
+    description = "Resource to be deleted",
+    required = true)
+  String resourceType;
+
+  @CommandLine.Option(
+    names = {"-id", "--identifiers"},
+    description = "Resource Identifiers if deleting a single resource",
+    required = true)
+  String identifiers;
+
+  @CommandLine.Option(
+    names = {"-at", "--access-token"},
+    description = "access token for fhir server")
+  String accessToken;
+
+  @Override
+  public void run() {
+    String url = baseUrl + "/" + resourceType + "?_id=" + identifiers + "&_cascade=delete";
```

Comment:
Can we make the cascades delete optional? maybe make it default not, considering the risk involved in data loss, so the person running the command has to actually opt in